cogs/wordfilter/wordfilter.py

```diff
@@ -550,20 +570,100 @@ async def on_message(self, msg):
+    @stat.command(name="rawusage")
+    @commands.guild_only()
+    @checks.mod_or_permissions(manage_messages=True)
+    async def rawCensorUsageList(self, ctx):
+        """
+        displays a raw usage list of all the censored words that have been used
+        """
+        user = ctx.message.author
+        rawUsageStats = await self.config.guild(ctx.guild).get_attr(KEY_USAGE_STATS)()
+
+        if rawUsageStats:
+            display = []
+            pageList = []
+            count = 1
+            for regex, timesUsed in rawUsageStats.items():
+                display.append(f"{count}. `{regex}` : `{timesUsed}`,")
+                count += 1
+            msg = "\n".join(display)
+            pages = list(chat_formatting.pagify(msg, page_length=400))
+            totalPages = len(pages)
+            totalEntries = len(display)
+
+            async for pageNumber, page in AsyncIter(pages).enumerate(start=1):
+                embed = discord.Embed(
+                    title=f"Filtered words for **{ctx.guild.name}**", description=page
+                )
+                embed.set_footer(text=f"Page {pageNumber}/{totalPages} ({totalEntries} entries)")
+                embed.colour = discord.Colour.red()
+                pageList.append(embed)
+            await menu(ctx, pageList, DEFAULT_CONTROLS)
+        else:
+            await user.send("Sorry you have no filtered words in **{}**".format(ctx.guild.name))
+
+    @stat.command(name="orderedusage")
+    @commands.guild_only()
+    @checks.mod_or_permissions(manage_messages=True)
+    async def orderedCensorUsageList(self, ctx):
+        """
+        displays an ordered usage list of all the censored words that have been used
+        """
+        user = ctx.message.author
+        rawUsageStats = await self.config.guild(ctx.guild).get_attr(KEY_USAGE_STATS)()
+
+        if rawUsageStats:
+            display = []
+            pageList = []
+            count = 1
+            for regex, timesUsed in dict(
+                reversed(sorted(rawUsageStats.items(), key=lambda item: item[1]))
+            ).items():
+                display.append(f"{count}. `{regex}` : `{timesUsed}`,")
+                count += 1
+            msg = "\n".join(display)
+            pages = list(chat_formatting.pagify(msg, page_length=400))
+            totalPages = len(pages)
+            totalEntries = len(display)
+
+            async for pageNumber, page in AsyncIter(pages).enumerate(start=1):
+                embed = discord.Embed(
+                    title=f"Filtered words for **{ctx.guild.name}**", description=page
+                )
+                embed.set_footer(text=f"Page {pageNumber}/{totalPages} ({totalEntries} entries)")
+                embed.colour = discord.Colour.red()
+                pageList.append(embed)
+            await menu(ctx, pageList, DEFAULT_CONTROLS)
+        else:
+            await user.send("Sorry you have no filtered words in **{}**".format(ctx.guild.name))
```

Comment:
This looks to be the exact same as the `rawusage` except with different sorting. What are your thoughts on factoring this out for both `rawusage` and `orderedusage`, and have both call into a common method? This way we don't have to repeat ourselves twice, e.g.
```py
    @stat.command(name="rawusage")
    async def rawCensorUsageList(self, ctx):
        await self.postUsageList(ctx)

    @stat.command(name="orderedusage")
    async def orderedCensorUsageList(self, ctx):
        await self.postUsageList(ctx, sorting=True)

    async def postUsageList(self, ctx, sorting=False):
        # Do stuff here
```